packages/alpinejs/src/magics/$model.js

```diff
@@ -0,0 +1,61 @@
+import { accessor } from '../interceptor'
+import { findClosest } from '../lifecycle'
+import { magic } from '../magics'
+import { reactive } from '../reactivity'
+
+magic('model', (el, { cleanup }) => {
+    let func = generateModelAccessor(el, cleanup)
+
+    Object.defineProperty(func, 'closest', { get() {
+        let func = generateModelAccessor(el.parentElement, cleanup)
```

Comment:
This is used by x-model so that there isn't an infinite loop when using `x-model="$model"`. So yah, it's the same, except it's only generated when needed and it references elements above the current one to prevent that infinite loop. I don't see a problem with it, do you?

Reply:
That does make sense!